### PR TITLE
Specify configuration file for POSTrain

### PR DESCRIPTION
**Pull request by b29308188**
issue #183

### CODE REVIEW

File: pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSConfigurator.java [hunk 1]
```diff
@@ -66,4 +68,8 @@ public ResourceManager getDefaultConfig() {
                         UNKNOWN_MODEL_PATH, UNKNOWN_LEX_PATH, BASELINE_LEX_PATH, MIKHEEV_LEX_PATH};
         return new ResourceManager(generateProperties(props));
     }
+
```
Comment (mssammon) [inline pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSConfigurator.java]: Why is this method needed (i.e. why not just inline "new ResourceManager(configFile)")?

File: pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSTrain.java [hunk 2]
```diff
@@ -36,14 +37,20 @@ public POSTrain() {
 
     public POSTrain(int iter) {
         this.iter = iter;
+        rm = new POSConfigurator().getDefaultConfig();
+        this.init();
+    }
+
+    public POSTrain(int iter, String configFile) throws IOException {
+        this.iter = iter;
+        rm = new POSConfigurator().getConfigFromFile(configFile);
```
Comment (mssammon) [inline pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSTrain.java]: This line should be "new POSConfigurator().getConfig( new ResourceManager( configFile ) )" -- this way, the user only needs to specify non-default values in the config.

File: pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSConfigurator.java (hunk 1 above)
Comment (b29308188) [inline pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSConfigurator.java]: We may want to have the default configuration even if there's no configuration file path specified in the POStrain.

**Comment by b29308188** [comment]
Modify POSTrain.java and remove getConfigFromFile() method in POSConfigurator.java.

**Pull request merged**